clicommand/annotate.go

```diff
@@ -108,7 +108,7 @@ var AnnotateCommand = cli.Command{
 
 		if cfg.Body != "" {
 			body = cfg.Body
-		} else if stdin.IsPipe() {
+		} else if terminal.IsTerminal(int(os.Stdin.Fd())) {
```

Comment:
Ah, is this the cross platform way to do it?

Reply:
Yup, or at least it let's it be someone else's problem. Our implementation didn't seem to work on Windows, despite it saying it did.